x-pack/solutions/search/packages/kbn-search-code-examples/src/api-tutorials/time_series_data_stream_tutorial.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+export const timeSeriesDataStreamTutorialCommands: string = `# Welcome to the tutorial on setting up time series data! 🎉
+# 🚀 This tutorial will guide you through setting up timeseries data stream, ingest sample data and run a query from the Kibana dev Console.
```

Comment:
How come "dev" is lower case and "Console" is uppercase? Should they both be lower case?